AltServer/AppDelegate.swift

```diff
@@ -44,7 +44,8 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     @IBOutlet private var installMailPluginMenuItem: NSMenuItem!
     @IBOutlet private var installAltStoreMenuItem: NSMenuItem!
     @IBOutlet private var sideloadAppMenuItem: NSMenuItem!
-    
+    @IBOutlet private var checkForUpdatesMenuItem: NSMenuItem!
```

Comment:
What exactly is this for? Is this adding a new menu item?

Reply:
yes, well replacing because an update to sparkle api